lib/ioki/apis/endpoints/create.rb

```diff
@@ -29,14 +29,14 @@ def full_path
       def call(client, model, args = [], options = {})
         outgoing_model_class = @outgoing_model_class || options[:outgoing_model_class] || model_class
 
-        unless model.is_a?(outgoing_model_class)
+        unless model.blank? || model.is_a?(outgoing_model_class)
```

Comment:
Could this lead to any issues at some point?

Reply:
Hope not :slightly_smiling_face: Do you have any specific scenario in mind?

There is a behavioral change: currently `passenger_client.create_my_model('my_id', my_model_object)` raises if `my_model_object` is `nil`.

With this PR it would no longer raise, but send an empty body (which most likely would get rejected).

I now changed it so that instead of checking for a `nil` model, it now checks whether the `outgoing_model_class` is `:empty`, which also defines the behavior more clearly.

Reply:
Sounds great! And no I had no specific scenario in mind. Just gut feeling 😄